src/pybind/mgr/dashboard/frontend/src/app/shared/services/crush.node.selection.service.ts

```diff
@@ -0,0 +1,142 @@
+import { AbstractControl } from '@angular/forms';
+
+import * as _ from 'lodash';
+
+import { CrushNode } from '../models/crush-node';
+
+export class CrushNodeSelectionService {
```

Comment:
I would suggest not using `Service` on the name of this class and filename, since this is not an `Angular Service` and its confusing to see a component extending from a `Service`.
And also move it to the `shared/classes` folder.

Reply:
Done :)